libs/platform/src/lib/components/form/select/select/select.component.ts

```diff
@@ -0,0 +1,254 @@
+                    'title', optionItem.querySelector('.fd-list__title').innerHTML);
+                optionItem.querySelector('.fd-list__title').setAttribute(
+                    'aria-label', optionItem.querySelector('.fd-list__title').innerHTML);
+                optionItem.querySelector('.fd-list__title').classList.add('fd-list__title--no-wrap');
+            }
+        });
+
+    }
+
+    /** @hidden */
+    ngAfterViewChecked(): void {
+        this._cd.detectChanges();
+    }
+
+
+    /** @hidden */
+    _setColumnsRatio(firstColumnRatio: number, secondColumnRatio: number): void {
+
+        let firstColumnProportion: number;
+        let totalProportions: number;
+        let secondColumnProportion: number;
+
+
+        totalProportions = firstColumnRatio + secondColumnRatio;
+        firstColumnProportion = Math.round(firstColumnRatio / totalProportions * 100);
+        secondColumnProportion = 100 - firstColumnProportion;
+
+
+        // setting option items
+        this.select._options.forEach((option) => {
+            const optionItem = option._getHtmlElement();
+            optionItem.querySelector('.fd-list__title').setAttribute(
+                'style', 'width: ' + firstColumnProportion + 'rem');
+            optionItem.querySelector('.fd-list__secondary').setAttribute(
+                'style', 'width: ' + secondColumnProportion + 'rem');
+        });
+
+    }
+
+    /**
+     * Method to emit change event
+     * @hidden
+     */
+    _emitChangeEvent<T>(modelValue: T): void {
+        const event = new FdpSelectionChangeEvent(this, modelValue);
+        this.selectionChange.emit(event);
+    }
+
+    /**
+     * Method to set selected item
+     * @hidden
+     */
+    _selectOptionItem(item: OptionItem): void {
+        if (this.mobile) {
+            this.selected = item;
+            this.value = item.label;
+            this.cd.detectChanges();
+
+            return;
+        }
+
+        this.value = item.label;
+        this._checkAndUpdate(item);
+        this.showList(false);
+    }
+
+    /**
+     * Method to set as selected
+     *  @hidden
+     */
+    _setAsSelected(item: OptionItem[]): void {
+        const selectedItem = item[0];
+
+        if (this._isSelectedOptionItem(selectedItem)) {
+            return;
+        }
+
+        this.selected = selectedItem;
+        this.value = this.displayValue(this.selected);
+      
+    }
+
+    /**
+     * Define is selected item selected
+     * @hidden
+     */
+    _isSelectedOptionItem(selectedItem: any): boolean {
+        return this.lookupKey && this.lookupValue(this.selected) === this.lookupValue(selectedItem) ||
+            this.displayValue(this.selected) === this.displayValue(selectedItem);
+    }
+
+    /**
+     * Define is selected item selected by display value
+     * @hidden
+     */
+    _isSelectedOptionItemByDisplayValue(selectedItem: any): boolean {
+        return this.selected && this.selected.label === selectedItem;
```

Comment:
```suggestion
        return this.selected?.label === selectedItem;
```

Reply:
> 
Can't do this change as "Expression expected error "was getting displayed on compilation